scripts/data/experiment.json

```diff
@@ -156,4 +156,26 @@
     "endDate": "2016-02-02",
     "lastEdited": "2016-02-04",
     "structure": []
+}, {
+    "id": "test9",
+    "title": "Lookit Test",
+    "description": "It's a test",
+    "state": "Active",
+    "eligibilityCriteria": "age >= 3 years and age < 6 years",
+    "structure": [
+        {
+            "type": "lookit-overview",
+            "title": "Learning from Others: overview",
+            "duration": "Ten minutes",
+            "whatHappens": "First your child will see two actors naming familiar objects. Sometimes they're correct, sometimes they're not. Then we'll show the same actors naming unfamiliar objects and ask your child some questions about which answers he or she trusts.",
+            "purpose": "How children track and use information about reliability.",
+            "subtitleLeft": "Familiar object videos",
+            "imgLeft": "imgLeft.png",
+            "captionLeft": "The 'Familiar object' videos look like this. Two actors will be introduced and asked to name various familar objects, like a brush or rubber duck. Usually, one is more accurate that the other.",
```

Comment:
Because this is human-focused text, please use quotation marks here instead of apostrophes.

`"The \"Familiar object\" videos..."`